units/host_allocator.go

```diff
@@ -310,7 +307,7 @@ func (j *hostAllocatorJob) setTargetAndTerminate(ctx context.Context, lenUpHosts
 		}
 		err := amboy.EnqueueUniqueJob(ctx, j.env.RemoteQueue(), NewHostDrawdownJob(j.env, drawdownInfo, utility.RoundPartOfMinute(1).Format(TSFormat)))
 		if err != nil {
-			grip.Error(errors.Wrap(err, "Error drawing down hosts"))
+			grip.Error(message.WrapError(err, message.Fields{"message": "Error drawing down hosts"}))
```

Comment:
nit: can this include the host allocator's job ID in the `message.Fields`? Typically, we include the job ID in log messages within amboy jobs for debugging purposes.